pkg/apis/troubleshoot/v1beta2/analyzer_shared.go

```diff
@@ -186,6 +186,13 @@ type CephStatusAnalyze struct {
 	Namespace     string     `json:"namespace" yaml:"namespace"`
 }
 
+type VeleroAnalyze struct {
+	AnalyzeMeta   `json:",inline" yaml:",inline"`
+	Outcomes      []*Outcome `json:"outcomes" yaml:"outcomes"`
+	CollectorName string     `json:"collectorName,omitempty" yaml:"collectorName,omitempty"`
+	Namespace     string     `json:"namespace" yaml:"namespace"`
```

Comment:
I think this needs to be removed as well

Reply:
good catch. ✅